proto/ligato/vpp/interfaces/interface.proto

```diff
@@ -380,4 +382,23 @@ message IPIPLink {
     string src_addr = 2;
     // Destination VTEP IP address
     string dst_addr = 3;
+}
+
+message WireguardLink {
+  uint32 user_instance = 1;
+
+  // Interface index
+  uint32 interface = 2;
```

Comment:
We DO NOT store interface index from VPP's runtime in the NB Proto API - the index is by definition only valid in the currently running session and might cause unexpected behaviour after VPP restart.

Change this to string so it refers to logical name of the interface and use the name-index mapping to store the reference between them.